readme.md

```diff
@@ -329,6 +339,24 @@ Default: `false`
 
 If `true`, no quoting or escaping of arguments is done on Windows. Ignored on other platforms. This is set to `true` automatically when the `shell` option is `true`.
 
+### child_process.kill([signal], [options])
+
+You may kill the child process with this function. It is slightly different that the original function.
+This one allows you to kill the process with `SIGTERM` and after a specified timeout (5 seconds) a `SIGKILL` is invoked to ensure killing the process.
+
+#### options.retry
+
+Type: `boolean`<br>
+Default: true
+
+Tells the kill to start a second attemp to kill.
+
+#### options.retryAfter
+
+Type: `string | number`<br>
+Default: 5000
+
+Specifies a timeout in ms. This one defines the delay until you try to kill the pocess again.
```

Comment:
"pocess" has a typo. "This one" might not be needed. Also "you try" implies the user needs to retry, but the retry is automatic.

```suggestion
How long to wait for the child process to terminate before sending `SIGKILL`.
```

Reply:
Fixed: 2fb0ae0a7664c4c240d3ae9f764b6b0976549827